perllib/FixMyStreet/Script/BANES/PassthroughConfirm.pm

```diff
@@ -0,0 +1,115 @@
+package FixMyStreet::Script::BANES::PassthroughConfirm;
+
+use Moo;
+use FixMyStreet::DB;
+use FixMyStreet::Queue::Item::Report;
+use FixMyStreet::SendReport::Open311;
+use Open311;
+
+has body => (
+    is => 'ro',
+    default => sub { FixMyStreet::DB->resultset('Body')->find( { name => 'Bath and North East Somerset Council' } ) or die $! }
+);
+
+sub send_reports {
+    my ($self) = @_;
+
+    my $problems = $self->_problems;
+
+    while (my $row = $problems->next) {
+        my $item = FixMyStreet::Queue::Item::Report->new( report => $row );
+        FixMyStreet::DB->schema->cobrand($item->cobrand);
+        my $confirm_id = $row->external_id;
+        $item->cobrand->set_lang_and_domain($row->lang, 1);
+        $item->_create_vars;
+        $item->h->{sending_to_banes_passthrough} = 1;
+        my $sender_info = {
+            method => 'Open311',
+            config => $self->body,
+        };
+        my $reporter = FixMyStreet::SendReport::Open311->new;
+        $reporter->add_body( $self->body, $sender_info->{config} );
+        $item->_set_reporters([$reporter]);
+        my $service_code = 'Passthrough-' . $row->contact->email . '@example.org';
```

Comment:
So if I've thought it through right, I think you can now remove the `@example.org` here (and no need for the if statement in your open311-adapter change to strip it off again, because the code is now using the `Passthrough-` prefix and that's all it needs?